### ISSUE TITLE
Aeotec Smart Power Strip gets stuck during interview in z-wave gateway (via Z-wave JS Mqtt via Home Assistant)

### ISSUE THREAD

**Issue by 130s** [kinu-garage/hut_10sqft#616]
Seen in https://github.com/130s/hut_10sqft/issues/610#issuecomment-982293756.

**Comment by 130s** [comment]
![image](https://user-images.githubusercontent.com/1840401/143991258-1e4aa428-1482-43d4-946f-a8d80ed7d752.png)
It once became green but later became yellow and for 10+ mins it's been like this.

What doesn't seem to help:
- Re-interview
- Press the "action button", and press it again.

**Comment by 130s** [comment]
I noticed that when I initiate the inclusion of the device in question, the power supply to all the devices that gets power from this device will be cut. That includes the ethernet hub that provides LAN to the HoA's server rpi3-1 is connected to, meaning when I trigger the z-wave inclusion, the HoA server loses the IP connection. Not sure if that's affecting but that certainly adds complexity.

So I relocated the power supply so that the hub doesn't lose power upon inclusion. But the same issue remains.

**Comment by 130s** [comment]
Not sure if https://github.com/130s/hut_10sqft/issues/187#issuecomment-593005413 was the sign that the device' z-wave connectivity to have started deteriorating.

Very similar to this issue [reddit.com](https://www.reddit.com/r/SmartThings/comments/c2oegv/aeotec_smart_power_strip_no_longer_pairs_with/) that was with SmartThings, not HoA. I don't know what "Are you using a custom device handler? If so, re-publish than handler and try again." means to me as I'm not using anything custom.

This video taken by 3rd party https://www.youtube.com/watch?v=XM3_ajitlbk shows pressing the Active button 3 times in a row, which is not included in the [official doc (aeotec.com)](https://help.aeotec.com/support/solutions/articles/6000165753-smart-power-strip-user-guide-).

**Comment by 130s** [comment]
Asked customer support the following.

----

Aeotec Product: - (the product in question not found in pulldown)
Gateway: GoControl HUSBZB-1. ZwaveJS MQTT via Home Assistant
Subject: Adding "Smart Power Strip" doesn't complete, device shown as "Dead"
Z-wave frequency: US 908.4, 908.42, 916 MHz
Description: The device in question is DSC11 https://www.amazon.com/Aeotec-Aeon-Labs-DSC11-Sockets/dp/B00H3RL6JW/ref=cm_cr_arp_d_product_top?ie=UTF8 (I couldn't find it on aeotec.com any more).

Adding the device from the gateway gets stuck indefinitely. I see `Protocolinfo` and `NodeInfo` steps are green. On ZwaveJS MQTT, "Status" is red and says "Dead".

A few years ago with a different gateway (Z-way on RPi2), I was able to finish inclusion. But I stopped using that gateway.

I did factory-resetting the device following https://help.aeotec.com/support/solutions/articles/6000165753-smart-power-strip-user-guide- but I still see the same issue.

**Comment by 130s** [comment]
Tech support is responsive https://help.aeotec.com/support/tickets/1466116 (I guess closed site). Not resolution yet though.

**Comment by 130s** [comment]
Got a response, and the tech support person was able to complete inclusion, with Zwave-JS! Incredible tech support already.

> One thing i would recommend, with older z-wave devices it may be good to perform a hard power cycle on this device:
> 1. Switch the black surge protection switch OFF
> 2. Wait 6 minutes
> 3. Power the Smart Strip back on and try unpairing and re-pairing again. Maybe move the switch closer to ensure it isn't a communication issue.

Didn't know there is a black surge switch. I turned it off, and I didn't wait for 6 mins today but only waited for 20 secs, then tried again. Unlike previous times e.g. https://github.com/130s/hut_10sqft/issues/616#issuecomment-982303364, this time I do see the z-wave was able to pull in some product info. But the inclusion is still stuck. 
![20211224015554_Aeotec_DSC11_inclusion](https://user-images.githubusercontent.com/1840401/147327206-2d6e8fc1-39d0-4d12-ae02-74bf4ea344ff.png)


I've removed the ID 29 device, factory-reset the device, then turn off the black surge switch, then call a day. Will try tomorrow.

**Comment by 130s** [comment]
Been failing.

Tried-1:
1. Factory-reset.
1. Turn off the surge black button.
1. Wait for 10+ hours.
1. Turn on the surge black button.
1. Start inclusion. -> Didn't even advance to device info recognized as https://github.com/130s/hut_10sqft/issues/616#issuecomment-1000688164

Tried-2: No factory-reset. This must be closer to what the tech support recommended.
1. Turn off the surge black button.
1. Wait for 10+ hours.
1. Turn on the surge black button.
1. Start inclusion. -> Didn't even advance to device info recognized as https://github.com/130s/hut_10sqft/issues/616#issuecomment-1000688164

**Comment by 130s** [comment]
Node 32 seems available, but re-interview immediately terminates with an error.
```
:
2021-12-27 22:24:25.327 INFO ZWAVE: Node 21: value updated: 50-0-value-66049 0 => 0
2021-12-27 22:24:25.967 INFO MQTT: MQTT client reconnecting
2021-12-27 22:24:25.974 ERROR MQTT: Mqtt client error connect ECONNREFUSED 127.0.0.1:1883
Error: connect ECONNREFUSED 127.0.0.1:1883
at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1161:16)
2021-12-27 22:24:25.982 INFO MQTT: MQTT client closed
2021-12-27 22:24:27.924 INFO ZWAVE: Node 32: interview started
2021-12-27 22:24:27.976 INFO ZWAVE: Node 32: interview stage PROTOCOLINFO completed
2021-12-27 22:24:27.978 DEBUG MQTT: Publishing to zwave/nodeID_32/status: { time: 1640643867978, value: false, status: 'Dead', nodeId: 32 } with options { qos: 1, retain: true }
2021-12-27 22:24:28.983 INFO MQTT: MQTT client reconnecting
2021-12-27 22:24:28.989 ERROR MQTT: Mqtt client error connect ECONNREFUSED 127.0.0.1:1883
Error: connect ECONNREFUSED 127.0.0.1:1883
at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1161:16)
2021-12-27 22:24:28.996 INFO MQTT: MQTT client closed
2021-12-27 22:24:30.157 DEBUG MQTT: Publishing to zwave/nodeID_28/50/0/value/66817: { time: 1640643870157, value: 0 } with options { qos: 1, retain: true }
2021-12-27 22:24:30.159 INFO ZWAVE: Node 28: value updated: 50-0-value-66817 0 => 0
2021-12-27 22:24:31.997 INFO MQTT: MQTT client reconnecting
:
at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1161:16)
2021-12-27 22:24:32.013 INFO MQTT: MQTT client closed
2021-12-27 22:24:35.014 INFO MQTT: MQTT client reconnecting
2021-12-27 22:24:35.019 ERROR MQTT: Mqtt client error connect ECONNREFUSED 127.0.0.1:1883
Error: connect ECONNREFUSED 127.0.0.1:1883
at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1161:16)
2021-12-27 22:24:35.027 INFO MQTT: MQTT client closed
2021-12-27 22:24:36.310 ERROR ZWAVE: Interview of node 32 has failed: The node is dead
2021-12-27 22:24:36.311 DEBUG MQTT: Publishing to zwave/nodeID_32/status: { time: 1640643876310, value: false, status: 'Dead', nodeId: 32 } with options { qos: 1, retain: true }
2021-12-27 22:24:38.028 INFO MQTT: MQTT client reconnecting
2021-12-27 22:24:38.034 ERROR MQTT: Mqtt client error connect ECONNREFUSED 127.0.0.1:1883
Error: connect ECONNREFUSED 127.0.0.1:1883
```

**Comment by 130s** [comment]
I'd rather wait for migrating HASS onto another OS https://github.com/130s/hut_10sqft/issues/644.

That said I took the log/config files from z-wave js.

<details><summary>json files</summary>

```
{
  "id": 40,
  "name": "",
  "loc": "",
  "values": [
    {
      "id": "40-134-0-libraryType",
      "nodeId": 40,
      "commandClass": 134,
      "commandClassName": "Version",
      "endpoint": 0,
      "property": "libraryType",
      "propertyName": "libraryType",
      "type": "number",
      "readable": true,
      "writeable": false,
      "label": "Library type",
      "stateless": false,
      "list": true,
      "states": [
        {
          "text": "Unknown",
          "value": 0
        },
        {
          "text": "Static Controller",
          "value": 1
        },
        {
          "text": "Controller",
          "value": 2
        },
        {
          "text": "Enhanced Slave",
          "value": 3
        },
        {
          "text": "Slave",
          "value": 4
        },
        {
          "text": "Installer",
          "value": 5
        },
        {
          "text": "Routing Slave",
          "value": 6
        },
        {
          "text": "Bridge Controller",
          "value": 7
        },
        {
          "text": "Device under Test",
          "value": 8
        },
        {
          "text": "N/A",
          "value": 9
        },
        {
          "text": "AV Remote",
          "value": 10
        },
        {
          "text": "AV Device",
          "value": 11
        }
      ],
      "value": 3,
      "lastUpdate": 1642794100291,
      "newValue": 3
    },
    {
      "id": "40-134-0-protocolVersion",
      "nodeId": 40,
      "commandClass": 134,
      "commandClassName": "Version",
      "endpoint": 0,
      "property": "protocolVersion",
      "propertyName": "protocolVersion",
      "type": "string",
      "readable": true,
      "writeable": false,
      "label": "Z-Wave protocol version",
      "stateless": false,
      "list": false,
      "value": "3.67",
      "lastUpdate": 1642794100296,
      "newValue": "3.67"
    },
    {
      "id": "40-134-0-firmwareVersions",
      "nodeId": 40,
      "commandClass": 134,
      "commandClassName": "Version",
      "endpoint": 0,
      "property": "firmwareVersions",
      "propertyName": "firmwareVersions",
      "type": "string[]",
      "readable": true,
      "writeable": false,
      "label": "Z-Wave chip firmware versions",
      "stateless": false,
      "list": false,
      "value": [
        "1.1"
      ],
      "lastUpdate": 1642794100299,
      "newValue": [
        "1.1"
      ]
    },
    {
      "id": "40-114-0-manufacturerId",
      "nodeId": 40,
      "commandClass": 114,
      "commandClassName": "Manufacturer Specific",
      "endpoint": 0,
      "property": "manufacturerId",
      "propertyName": "manufacturerId",
      "type": "number",
      "readable": true,
      "writeable": false,
      "label": "Manufacturer ID",
      "stateless": false,
      "min": 0,
      "max": 65535,
      "list": false,
      "value": 134,
      "lastUpdate": 1642688395626,
      "newValue": 134
    },
    {
      "id": "40-114-0-productType",
      "nodeId": 40,
      "commandClass": 114,
      "commandClassName": "Manufacturer Specific",
      "endpoint": 0,
      "property": "productType",
      "propertyName": "productType",
      "type": "number",
      "readable": true,
      "writeable": false,
      "label": "Product type",
      "stateless": false,
      "min": 0,
      "max": 65535,
      "list": false,
      "value": 3,
      "lastUpdate": 1642688395630,
      "newValue": 3
    },
    {
      "id": "40-114-0-productId",
      "nodeId": 40,
      "commandClass": 114,
      "commandClassName": "Manufacturer Specific",
      "endpoint": 0,
      "property": "productId",
      "propertyName": "productId",
      "type": "number",
      "readable": true,
      "writeable": false,
      "label": "Product ID",
      "stateless": false,
      "min": 0,
      "max": 65535,
      "list": false,
      "value": 11,
      "lastUpdate": 1642688395633,
      "newValue": 11
    }
  ],
  "groups": [],
  "neighbors": [
    1,
    16,
    21,
    35,
    36,
    39
  ],
  "ready": false,
  "available": false,
  "hassDevices": {},
  "failed": false,
  "inited": false,
  "hexId": "0x0086-0x0003-0x000b",
  "dbLink": "https://devices.zwave-js.io/?jumpTo=0x0086:0x0003:0x000b:0.0",
  "manufacturerId": 134,
  "productId": 11,
  "productType": 3,
  "deviceConfig": {
    "filename": "/var/snap/zwavejs2mqtt/325/.ext-config/devices/0x0086/dsc11.json",
    "isEmbedded": true,
    "manufacturer": "AEON Labs",
    "manufacturerId": 134,
    "label": "DSC11",
    "description": "Smart Strip",
    "devices": [
      {
        "productType": 3,
        "productId": 11
      }
    ],
    "firmwareVersion": {
      "min": "0.0",
      "max": "255.255"
    },
    "associations": {},
    "paramInformation": {
      "_map": {}
    }
  },
  "productLabel": "DSC11",
  "productDescription": "Smart Strip",
  "manufacturer": "AEON Labs",
  "firmwareVersion": "1.1",
  "protocolVersion": 3,
  "nodeType": 1,
  "endpointsCount": 0,
  "endpointIndizes": [],
  "isSecure": false,
  "security": "None",
  "supportsSecurity": false,
  "supportsBeaming": true,
  "isControllerNode": false,
  "isListening": true,
  "isFrequentListening": false,
  "isRouting": true,
  "keepAwake": false,
  "maxDataRate": 40000,
  "deviceClass": {
    "basic": 4,
    "generic": 16,
    "specific": 1
  },
  "deviceId": "134-11-3",
  "status": "Dead",
  "interviewStage": "NodeInfo",
  "statistics": {
    "commandsTX": 45,
    "commandsRX": 117,
    "commandsDroppedRX": 0,
    "commandsDroppedTX": 17,
    "timeoutResponse": 23
  },
  "lastActive": 1643077501763,
  "healProgress": "skipped",
  "_name": "NodeID_40",
  "lastTransmit": 1643077501763,
  "errorReceive": false,
  "errorTransmit": true
}
```

</details>

If new environment doesn't solve then will ask on Z-Wave JS community. Found something related to connecting legacy device https://zwave-js.github.io/zwavejs2mqtt/#/device-support/polling but I'm not familliar with `polling`.

**Comment by 130s** [comment]
Miracle happened? It now connected back to HASS and now I can turn on/off an entity from HASS.

What I did?
1. I saw it became "`Unknown manufacturer 0xXXXX`" (similar as https://github.com/zwave-js/node-zwave-js/discussions/4705).
1. Physically pressed the power? button so that all power outlets on it turned the blue lights on on each slot.
1. Still physically sitting less than 30 cm away from the Z-Wave hub antenna.
1. Z-Wave "Heal"ed on `zwave-js`, which turned green.
1. Re-interviewed.
